package.json

```diff
@@ -22,16 +22,23 @@
         "release": "npm run coverage && npm run lint && npm run format-check"
     },
     "dependencies": {
-        "ask-sdk-core": "^2.10.1",
-        "ask-sdk-local-debug": "^1.0.0",
-        "ask-sdk-model": "^1.28.0",
-        "ask-smapi-model": "^1.5.0",
+        "ask-sdk-core": "^2.10.2",
+        "ask-sdk-local-debug": "^1.1.0",
+        "ask-sdk-model": "^1.35.1",
+        "ask-smapi-model": "^1.15.3",
         "chai": "^4.2.0",
         "debug": "^4.1.1",
-        "i18next": "^19.5.4",
+        "i18next": "^19.9.2",
         "lodash": "^4.17.15",
         "tslib": "^2.0.0"
     },
+    "peerDependencies": {
+        "ask-sdk-core": "^2.10.2",
+        "ask-sdk-model": "^1.35.1",
+        "ask-smapi-model": "^1.15.3",
+        "i18next": "^19.5.4",
+        "lodash": "^4.17.15"
```

Comment:
I was expecting all our dependencies to be peer dependencies (and dev-dependencies to assist with dev-box setup).  Our customers could be using specific versions of tslib, chai, debug for some reason whereas we can be flexible.

Reply:
Sure makes sense, updated to have all dependencies as peer dependencies in our package.json